html/canvas/element/manual/imagebitmap/createImageBitmap-resolves-in-task.any.js

```diff
@@ -0,0 +1,117 @@
+// META: title=createImageBitmap resolves in a task
+// META: script=/common/media.js
+// META: script=./common.sub.js
+function makeWorkerImageBitmap() {
+    return makeOffscreenCanvas().then(canvas => {
+        return createImageBitmap(canvas);
+    });
+}
+var imageSourceTypes = self.GLOBAL.isWorker() ?
+    [
+        { name: 'an OffscreenCanvas', factory: makeOffscreenCanvas },
+        { name: 'an ImageData', factory: makeImageData },
+        { name: 'an ImageBitmap', factory: makeWorkerImageBitmap },
+        { name: 'a Blob', factory: makeBlob('/images/pattern.png') },
+    ] :
+    imageSourceTypes;
```

Comment:
Something is causing the `HTMLVideoElement` cases to propagate uncaught exceptions to the global handler: https://wpt.fyi/results/html/canvas/element/manual/imagebitmap/createImageBitmap-resolves-in-task.any.html?diff&filter=ADC&run_id=6296029702127616&run_id=5153843161858048

Even weirder, the rejection value is an event.

Reply:
Yes, these sources already fail in other tests using that same helper, e.g. https://wpt.fyi/results/html/canvas/element/manual/imagebitmap/createImageBitmap-invalid-args.html?label=experimental&label=master&aligned I'm not quite sure why, to be honest.

Reply:
I worry that without debugging those, we won't get the chrome-dev-stability checker to pass, which will make merging require admin permissions. (And the admins might not allow it.)

Reply:
Hmm, although I see they are preexisting manual tests (!?) that are probably not made worse by this change. So the admins will probably allow it.